src/hosting/api.ts

```diff
@@ -449,8 +449,8 @@ export async function listVersions(site: string): Promise<Version[]> {
     const res = await apiClient.get<ListVersionsResponse>(`projects/-/sites/${site}/versions`, {
       queryParams,
     });
-    versions.push(...res.body.versions);
-    pageToken = res.body.nextPageToken;
+    versions.push(...(res.body?.versions || []));
```

Comment:
Body isn't the problem, it's the lack of `versions`.

I'd suggest changing `ListVersionsResponse` to have `versions?: Version[]`, then do `versions.push(...(res.body.versions ?? []));` here.

Similar to the `sites` response, if you really want to do that too.

Please add tests.

Reply:
Yeah, okay. I just did it. haha